pom.xml

```diff
@@ -1578,7 +1578,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <swagger-core-version>2.1.2</swagger-core-version>
         <swagger-parser-groupid>io.swagger.parser.v3</swagger-parser-groupid>
-        <swagger-parser-version>2.0.26</swagger-parser-version>
+        <swagger-parser-version>2.0.28-SNAPSHOT</swagger-parser-version>
```

Comment:
I think we will need to wait for the 2.0.28 release before merging this enhancement.

Reply:
It seems that [2.1.12](https://mvnrepository.com/artifact/io.swagger.parser.v3/swagger-parser/2.1.12) has already been released